src/javascripts/components/blog/ArticlePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+class ArticlePreview extends React.Component {
+  render() {
+    return (
+      <Link to="/article"><div className="articlePreview">
+        <div className="--imageContainer">
+          <img
+            src="./images/alice-in-wonderland.png"
+            alt="Picture of 'Author'"
+          />
+        </div>
+          <div className="--preview">
+            <h1>Down The Rabbit Hole</h1>
```

Comment:
This shouldn't be an `h1`, right?  It's not the title of the page in this instance, right?